### PR TITLE
feat: platform starter kit

### LINKED ISSUE #14290: [CAL-3338] platform starter kit
image

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3338](https://linear.app/calcom/issue/CAL-3338/platform-starter-kit)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
image

built with create-t3-app

- nextjs
- sqlite
- next auth
- cal atoms

### CODE REVIEW

File: apps/web/components/eventtype/AIEventController.tsx
Comment (p6l-richard) [inline apps/web/components/eventtype/AIEventController.tsx:1]: what's that? @PeerRich

Comment (p6l-richard) [reply apps/web/components/eventtype/AIEventController.tsx:1]: should we keep that?

Comment (PeerRich) [reply apps/web/components/eventtype/AIEventController.tsx:1]: a mistake. removing

**Comment by PeerRich** [comment]
once this is merged i need this:

CleanShot 2024-04-12 at 17 31 56@2x

**Pull request closed without merging**